script.simkl/service.py

```diff
@@ -16,12 +16,12 @@
 autoscrobble = __addon__.getSetting("autoscrobble")
 def getstr(strid): return __addon__.getLocalizedString(strid)
 
-#try:
-compdatefile = os.path.join(__addon__.getAddonInfo("path").decode("utf-8"), "resources/data/compdate.txt")
-with open(xbmc.translatePath(compdatefile), "r") as f:
-    __compdate__ = f.read()
-#except:
-#    __compdate__ = "ERROR: No such file or directory"
+try:
+    compdatefile = os.path.join(__addon__.getAddonInfo("path").decode("utf-8"), "resources/data/compdate.txt")
```

Comment:
can you please use join for all the nodes? `os.path.join(.........,"resources","data","compdate.txt")`